lib/bridge/IrcHandler.js

```diff
@@ -953,4 +970,27 @@ IrcHandler.prototype._setMatrixRoomAsInviteOnly = function(room, isInviteOnly) {
     );
 };
 
+IrcHandler.prototype.incrementMetric = function(metric) {
+    if (this._callCountMetrics[metric] === undefined) {
+        this._callCountMetrics[metric] = 0;
+    }
+    this._callCountMetrics[metric]++;
+}
+
+IrcHandler.prototype.getMetrics = function() {
+    const metrics = Object.assign({}, this._callCountMetrics);
+    this._callCountMetrics = {
+        "join.names": 0,
+        "join": 0,
+        "part": 0,
+        "pm": 0,
+        "invite": 0,
+        "topic": 0,
+        "message": 0,
+        "kick": 0,
+        "mode": 0,
```

Comment:
Extra ,

Reply:
[it's a personal preference thing](https://medium.com/@nikgraf/why-you-should-enforce-dangling-commas-for-multiline-statements-d034c98e36f8)